src/components/Menu/MenuItem/MenuItemButton/MenuItemButton.tsx

```diff
@@ -46,25 +50,51 @@ export const MenuItemButton: FC<MenuItemButtonProps> = ({
     },
   ]);
 
+  const handleOnClick = (
+    event: React.MouseEvent<HTMLButtonElement | MouseEvent>
+  ): void => {
+    if (disabled) {
+      event.preventDefault();
```

Comment:
Just a little curious, what would happen if it is disabled and we dont prevent default here?